apps/microtvm/reference-vm/zephyr/base-box/base_box_test.sh

```diff
@@ -37,3 +37,5 @@ if [ $board == "stm32f746xx" ]; then
 else
     pytest tests/micro/zephyr/test_zephyr_aot.py --zephyr-board=${board}
 fi
+
+pytest tests/micro/zephyr/test_zephyr_armv7m.py --zephyr-board=${board}
```

Comment:
curious if you are including CMSIS-NN anywhere?

Reply:
Not now. It seems we need download subset of CMSIS inside the test temporary folder to be able to build test project. Isn't so?
Another option is to eliminate need of CMSIS inside gemm C code implementation..

Reply:
it's tempting to inline `read_and_pad`, which should get inlined by the compiler at codegen time. CMSIS 5 is [under ASF 2.0](https://github.com/ARM-software/CMSIS_5/blob/develop/LICENSE.txt). the tricky thing is that there are quite a lot of header dependencies designed to support a variety of toolchains. it would be nice to preserve support for those.

given that, i'm tempted to opt for downloading CMSIS to the temporary directory. we will likely need to come up with a solution for this more broadly to work with CMSIS-NN as well cc @ashutosh-arm @u99127 @manupa-arm

Reply:
Ok. Will download subset of CMSIS headers in temporary directory for this specific test.

Reply:
Done.